File: JAVASCRIPT/ClassObject/OOPS/Abstraction.js
class KuchKuch{
  #Substrction(a,b){
    return b-a;
  }
  CallMe(a,b){
    return this.#Substrction(a,b);
  }
}

let obj=new KuchKuch();
let ans=obj.CallMe(10,11);
console.log(ans);

File: Html_Css_Js/JAVASCRIPT/ClassObject/Abstraction.cpp
#include<bits/stdc++.h>
using namespace std;

class Abstraction{
  private:
  int Multiply(int a,int b){
    return a*b;
  }
  public:
  int CallMe(int a, int b){
    return this->Multiply(a,b);
  }
};
int main(){
  Abstraction obj;
  cout<<obj.CallMe(10,10);
}